### DISCUSSION TITLE
How to change the indentation of code

### DISCUSSION THREAD

**Discussion by nianfouyi** [LazyVim/LazyVim#953] [Q&A]
Hi everyone, I'm using lazyvim, I have a question, how to change the indent line, I know to use "<" or ">" in vim, but the keyboard doesn't work in lzayvim, I found that I can use "<<" or ">>" does the same job, I want to know why, can anyone help me?

**Comment by dpetka2001** [comment]
I just tried `>` and it works as expected. However, like `:h >` mentions it also expects a `motion` after `>`. So you have to give it one. For example if your cursor is on the first of 2 consecutive lines, if you press `>j` it will indent both lines to the right.
